<?xml version="1.0" encoding="utf-8"?>
<test-results date="2018-10-18" name="Expecto.Tests" total="4" errors="0" failures="1" ignored="0" not-run="0" inconclusive="0" skipped="0" invalid="0" time="14:04:04">
  <environment expecto-version="8.4.2" clr-version="4.0.30319.42000" os-version="Microsoft Windows NT 6.2.9200.0" platform="Linux" cwd="C:\Repositories\FisherSMPD\src\Fisher.Common.Tests" machine-name="DIT299" user="Dominik.Kotecki" user-domain="ROSSPOL" />
  <culture-info current-culture="pl-PL" current-uiculture="pl-PL" />
  <test-suite type="Assembly" name="Expecto.Tests" executed="True" result="Failure" success="False" time="0.525" asserts="0">
    <results>
      <test-case name="Database/load file/Test1" executed="True" result="Failure" success="False" time="0.203" asserts="0">
        <failure>
          <message>
. Actual value was 3213321 but had expected it to be 64.
  C:\Users\Anthony Lloyd\src\expecto\Expecto\Expect.fs(287,1): Expecto.Expect.equal@287-8.Invoke(String x)
</message>
        </failure>
      </test-case>
      <test-case name="Probability/getPossibleCombinations/Test combinations" executed="True" result="Success" success="True" time="0.048" asserts="0" />
      <test-case name="Fisher/S/sa" executed="True" result="Success" success="True" time="0.094" asserts="0" />
      <test-case name="Fisher/average vector/Test1" executed="True" result="Success" success="True" time="0.088" asserts="0" />
    </results>
  </test-suite>
</test-results>